src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int PARAM_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeWtoFile(createReport(readFromFile(fromFileName)), toFileName);
+    }
+
+    public List<String> readFromFile(String fromFileName) {
+        List<String> fileLines;
+        try {
+            fileLines = Files.readAllLines(Paths.get(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
+        }
+        return fileLines;
+    }
+
+    public int[] createReport(List<String> fileLines) {
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/81ce4ad8-6d68-47ff-b31e-03f7493b769a)

This is not implemented. **createReport** method must return String (or List<String>) report. Not strange int array.

Reply:
I changed it to "List" but I don't understand what's strange about an int array.